pkg/client/daemon/dbus/resolved.go

```diff
@@ -1,21 +1,19 @@
 package dbus
 
 import (
+	"context"
 	"errors"
 	"fmt"
 	"net"
 	"strings"
 	"syscall"
 
 	"github.com/godbus/dbus/v5"
+
+	"github.com/datawire/dlib/dlog"
 )
 
 type (
-	// A ResolveD is a `dbus.Conn` with methods to communicate with the ResolveD daemon.
```

Comment:
I think this is mostly bc of my ignorance and lack of experience with resolveD, but are we not using a persistent connection to increase stability (e.g. and not have some failure if the connection is closed unexpectedly) and bc the number of calls are infrequent enough that it's not problematic to make a new connection each time we need one?

Reply:
The calls are very infrequent, yes. Only happens when we update dns (intercept a new namespace) or when the set of IP subnet changes (which in reality almost never happens), so I saw no reason why the connection should be kept open.